install-gnome-themes

```diff
@@ -73,9 +73,10 @@ if [ "$REQUIREMENTS" == 1 ]; then
 
 	if [ "$OS" == 'Debian' ] ||
            [ "$OS" == 'Ubuntu' ] ||
+	   [ "$OS" == 'SPERIX-OS' ] ||
            [ "$OS" == 'LinuxMint' ]; then
 		# Ubuntu GNOME comes with Numix (for 3.18) preinstalled, we will remove it to avoid confusion
-		sudo apt remove numix-gtk-theme
+		sudo apt remove numix-gtk-theme -y
```

Comment:
I think it's a bad idea to do this automatically -- let the user decide if they're OK with this installation.